requirements.txt

```diff
@@ -1 +1,9 @@
-pytest
\ No newline at end of file
+pytest
```

Comment:
These all need to install in this PR modify?

Reply:
Yes, what package do you want to delete?

Reply:
I want to make sure this list is for docker container or our dev env?

Reply:
I remove the requirements